src/main/java/org/testng/internal/Invoker.java

```diff
@@ -1609,6 +1609,21 @@ public boolean isTrue(ITestNGMethod m, IClass c) {
     return result;
   }
 
+  private boolean isConfiguration(ITestNGMethod method) {
```

Comment:
Take the time to use this method for `InvokedMethod#isConfigurationMethod` too.

Then, rename the method to `isConfigurationMethod`.